docs/release/release_notes/release-0.3.11.rst

```diff
@@ -0,0 +1,7 @@
+==============
+Release 0.3.11
+==============
+
+* Fix a regression from 0.3.10 that caused wrong BlockLocations to be returned in CachingFileSystem#listLocatedStatus
+
```

Comment:
nit: Extra line

Reply:
In future, we may also start segregating some improvements based on the engine.

Reply:
Ya, release note page is open form and we can add anything. This change is to just introduce it.